home/forms/search.py

```diff
@@ -119,6 +125,8 @@ class SearchForm(forms.Form):
     clear_filter = forms.BooleanField(initial=False, required=False)
     clear_label = forms.BooleanField(initial=False, required=False)
 
+    tags = forms.MultipleChoiceField(choices=get_tags, required=False)
```

Comment:
Tags is configured as multiple choice however selecting additional tags currently overwrites the former

Reply:
I suppose it might be confusing - I think you're right and whilst the tag filter is only available via the link and always does a single choice this form choice should reflect that

Reply:
after our discussion gonna change this to have tag filters stack up, i.e. remain multiplechoice

Reply:
clarified in standup - the current behaviour is as expected at this stage